modules/op-tkn-hub-interaction.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// *  cli_reference/tkn_cli/op-tkn-reference.adoc
+
+[id="op-tkn-hub-interaction_{context}"]
+= Hub interaction commands
+
+Interact with Tekton Hub for resources such as tasks and pipelines.
+
+== hub
+Interact with hub.
+
+.Example: Display help
+[source,terminal]
+----
+$ tkn hub -h
+----
+
+.Example: Interact with hub API server
+[source,terminal]
+----
+$ tkn hub --api-server https://api.hub.tekton.dev
+----
+
+[NOTE]
+====
+For each command examples given below, you can get the corresponding sub-commands and flags by executing `tkn hub <command> --help`.
+====
+
+== hub downgrade
+Downgrade an installed resource
+
+.Example: Downgrade an installed resource
+[source,terminal]
+----
+$ tkn hub downgrade task -n mynamespace --to <resourceVersion>
+----
+
+== hub get
+Get resource manifest by its name, kind, catalog, and version.
+
+.Example: Get resource manifest
+[source,terminal]
+----
+$ tkn hub get [pipeline | task] --from tekton --version <resourceVersion>
+----
+
+== hub info
+Display information about a resource by its name, kind, catalog, and version.
+
+.Example: Display resource information
```

Comment:
Display resource information of latest version of task `mytask` from `tekton` catalog

Reply:
@sm43 whats wrong with using `--version`? Anyway we are using `--version` in previous commands

Reply:
Mentioning `mytask` and `tekton` in the description would be too specific; I think the description should be generic as it is, and the example should contain the specifics.

Reply:
just to clarify 😅 all my comments were for examples not for command description
```
 Example: Display resource information of latest version of task `mytask` from `tekton` catalog
 ```